docs/faq.md

```diff
@@ -1,5 +1,14 @@
 # FAQ
 
+**I did a fresh `git clone` of the EarlGreyExample CocoaPods Demo project followed by a
```

Comment:
He plans on getting it in for the next release so lets remove this? we might forget to not remove it.